Services/Certificate/classes/File/Background/classs.ilCertificateBackgroundImageDelete.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * @author  Niels Theen <ntheen@databay.de>
+ */
+class ilCertificateBackgroundImageDelete
+{
+    public function deleteBackgroundImage(string $version)
+    {
+        $result = true;
+        if (file_exists($this->getBackgroundImageThumbPath())) {
+            $result = $result & unlink($this->getBackgroundImageThumbPath());
```

Comment:
Can you use a `&&` here and below?

Reply:
I suggest we go a step further and remove the return and therefor the `$result`. I don't see that this is ever needed.